engine/game/index.ts

```diff
@@ -44,6 +44,7 @@ export class Game extends EventBus {
 	public start() {
 		this.scheduler.start();
 		this.scheduler.subscribe('update.logic', (dt) => {
+			this.input.capture();
```

Comment:
You don't want to put that here, since capture actually registers the event. So you're re-adding the event listeners every frame. Massive memory leaks and probably lots and lots of performance problems. Move it up to outside of that callback. :)